python/docs/source/getting_started/install.rst

```diff
@@ -96,9 +96,7 @@ If you want to make Spark Connect default, you can install and additional librar
 It will automatically install ``pyspark`` library as well as dependencies that are necessary for Spark Connect.
 If you want to customize ``pyspark``, you need to install ``pyspark`` with the instructions above in advance.
 
-Note that ``pyspark`` command will use ``--master`` option for Spark Connect remote URL instead of ``--remote`` option.
-
-See also `Quickstart: Spark Connect <quickstart_connect.html>`_ for how to use it.
+This package supports both ``spark.master`` (``--master``) with a locally running Spark Connect server, and ``spark.remote`` (``--remote``) including ``local*`` as well as connection URIs such as ``sc://localhost``. See also `Quickstart: Spark Connect <quickstart_connect.html>`_ for how to use it.
```

Comment:
Does `local*` mean `local[K,F]` syntax in this line, @HyukjinKwon ? If then, it looks a little confusing to me.

Reply:
Actually I also meant local-cluster too :-)..

Reply:
let me fix a little bit more

Reply:
Thank you!